lib/services/ios-project-service.ts

```diff
@@ -219,6 +229,13 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
     <string>${options.teamID}</string>
 `;
 		}
+		if (options && options.provision) {
+			plistTemplate += `    <key>provisioningProfiles</key>
```

Comment:
The time is tight, let's do this in a separate PR if we ever touch these two again.